plan/logical_plan_builder.go

```
@@ -618,10 +618,7 @@ func (a *havingAndOrderbyExprResolver) Leave(n ast.Node) (node ast.Node, ok bool
 				}
 			}
 		} else {
-			index, a.err = a.resolveFromSchema(v, a.p.GetSchema())
-			if a.err != nil {
-				return node, false
-			}
+			index, _ = a.resolveFromSchema(v, a.p.GetSchema())
```

add comment about ignore the error